### ISSUE TITLE
Failed to update from version control when running Pootle from a checkout

### ISSUE THREAD

**Issue by julen** [translate/pootle#1547]
I'm testing Pootle from a git-svn checkout as usual.

Then I went to try the support for Mercurial integration, did a checkout from the command line, files appeared on the web interface, but after clicking "Update all from version control", Pootle thinks my project's VCS is git rather than Mercurial.

This is what the console reports (the same for each file):
2010-07-29 09:06:06,677 ERROR near fatal catastrophe, exception [GIT] checkout failed (['git', '--git-dir', u'/home/julen/dev/translate/Pootle/.git', 'checkout', u'po/fxtest/eu/toolkit/chrome/global/nsTreeSorting.properties']): error: pathspec 'po/fxtest/eu/toolkit/chrome/global/nsTreeSorting.properties' did not match any file(s) known to git.

So basically Pootle is getting the VCS information from the top source dir rather than from the translation project specific dir.
